src/PowerShellEditorServices/Server/PsesDebugServer.cs

```diff
@@ -34,9 +36,10 @@ public class PsesDebugServer : IDisposable
             _loggerFactory = factory;
             _inputStream = inputStream;
             _outputStream = outputStream;
+            _serverStart = new TaskCompletionSource<bool>();
```

Comment:
```suggestion
            _serverStarted = new TaskCompletionSource<bool>();
```

Reply:
gonna change this to `_serverRunning` as it will be used to "wait until the server shuts down"

Reply:
changed it to that because when the "result is set" it's like saying set the result of the "running". Let me know if you feel stronger about `_serverStopped`